rocketc/rocketc.py

```diff
@@ -43,22 +46,34 @@ class RocketChatXBlock(XBlock, XBlockWithSettingsMixin, StudioEditableXBlockMixi
     )
 
     default_channel = String(
-        display_name="Default Channel",
-        default="",
+        display_name="Specific Channel",
         scope=Scope.content,
         help="This field allows to select the channel that would be accesible in the unit",
         values_provider=lambda self: self.get_groups(),
     )
-    default_group_enable = Boolean(
+
+    ui_is_block = Boolean(
```

Comment:
Please specify this var refers to when the rocket chat view is the default web view